static/templates/about_zulip.hbs

```diff
@@ -1,20 +1,20 @@
 <div id="about-zulip" class="overlay flex new-style" tabindex="-1" role="dialog" data-overlay="about-zulip" aria-hidden="true">
     <div class="overlay-content modal-bg">
-        <button type="button" class="exit" aria-label="{{ _('Close') }}"><span aria-hidden="true">&times;</span></button>
+        <button type="button" class="exit" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
         <div class="modal-body">
             <div class="about-zulip-logo">
-                <a target="_blank" rel="noopener noreferrer" href="https://zulip.com"><img src="/static/images/logo/zulip-org-logo.svg" alt="{{ _('Zulip') }}" /></a>
+                <a target="_blank" rel="noopener noreferrer" href="https://zulip.com"><img src="/static/images/logo/zulip-org-logo.svg" alt="{{t 'Zulip' }}" /></a>
             </div>
             <p>
                 <strong>Zulip Server</strong>
                 <br />
                 Version {{zulip_version}}
-                <i class="fa fa-copy tippy-zulip-tooltip" data-tippy-content="Copy version" data-tippy-placement="right" data-clipboard-text="{{zulip_version}}"></i>
-                {% if zulip_merge_base and zulip_version != zulip_merge_base %}
+                <i class="fa fa-copy tippy-zulip-tooltip" data-tippy-content="Copy version" data-tippy-placement="right" data-clipboard-text="{zulip_version}"></i>
+                {{#if (and zulip_merge_base (not (eq zulip_version zulip_merge_base)))}}
```

Comment:
Let's move this conditional into the caller; we try to avoid complex logic in handlebars templates.